erpnext/patches/v7_2/update_salary_slips.py

```diff
@@ -0,0 +1,17 @@
+import frappe
+from erpnext.hr.doctype.process_payroll.process_payroll import get_month_details
+
+def execute():
+	salary_slips = frappe.db.sql("""select fiscal_year, month, name from `tabSalary Slip` 
+				where (month is not null or month != '') 
+				and (fiscal_year is not null or fiscal_year != '') and
+				(start_date is null  or start_date = '') and 
+				(end_date is null  or end_date = '') and docstatus != 2 and 
+				posting_date between '2016-01-01' and '2016-06-30'""")
```

Comment:
Why this date conditions? There can be such salary slips beyond this period, depends on deployment date.